app/controllers/schools_controller.rb

```diff
@@ -166,40 +166,34 @@ def authorized_students_for_overview(school)
   end
 
   # Methods for Dashboards #
+  def shared_student_fields(student)
```

Comment:
@edavidsonsawyer For endpoints like this, where we're sending down a lot of events, using `as_json(only: [:foo, :bar])` to only serialize what the UI needs can save a lot of bytes!